.github/workflows/update-and-prepare-release.yml

```diff
@@ -0,0 +1,52 @@
+# TODO abort if there are no changes to the Gemfile.
```

Comment:
What is this workflow used for? If it's copied from another repo where I wrote it, it's probably not necessary.

Reply:
It came back from the original repo, but was actually never connected up to a script

Ie `script/workflows/prepare-release.sh` this path doesn't exist that its actually calling, so will pull this !

Reply:
![image](https://github.com/pact-foundation/pact-ruby-cli/assets/19932401/4f78df89-58d3-4a9c-9535-5957b6599501)